projects/core/src/i18n/i18n.module.ts

```diff
@@ -17,13 +17,8 @@ import { CxNumericPipe } from './numeric.pipe';
 
 export function initI18nConfig(
   configInitializer: I18nConfigInitializer,
-  config: I18nConfig,
-  featureConfigService: FeatureConfigService
+  config: I18nConfig
 ): ConfigInitializer | null {
-  // TODO(#11515): remove it in 4.0
-  if (!featureConfigService.isLevel('3.2')) {
-    return null;
-  }
```

Comment:
also need to remove `FeatureConfigService` from `deps: [I18nConfigInitializer, I18nConfig, FeatureConfigService],`.